.github/ISSUE_TEMPLATE/problem.yml

```diff
@@ -0,0 +1,115 @@
+name: Problem
+description: Create a report to help us improve
+labels: [bug, triage]
+
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: Describe the problem
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    id: template
+    attributes:
+      label: Template
+      description: Many problems are related to a specific Copier template. If you can provide a template, please do it. It can be a Git repo URL, a .zip file containing the template, or instructions to build it.
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. Run `copier ...`
+        2. Answer ...
+        3. Choose ...
+        4. See error
+    validations:
+      required: true
```

Comment:
Maybe the actual situation description is good enough, so no need to require this IMHO.

```suggestion
```